src/ticket.c

```diff
@@ -299,7 +312,12 @@ int acquire_ticket(struct ticket_config *tk, cmd_reason_t reason)
 		return RLT_EXT_FAILED;
 	}
 
-	rv = new_election(tk, local, 1, reason);
+	if (tk->mode == TICKET_MODE_MANUAL) {
+		rv = manual_selection(tk, local, 1, reason);
```

Comment:
The reason can only be one, right?

Reply:
I was thinking about two cases: manual grant or reacquiring a ticket that used to belong to this site but it has been restarted.

Reply:
I see. But isn't it that even on reacquire it's still the manual ticket, so the rules for manual tickets apply.